tests/test_icrestimator.py

```diff
@@ -82,6 +102,79 @@ def test_estimate_lambda():
     # assert np.allclose(desired_lmda, lmda_e.T, atol=tolerance)
 
 
+@pytest.mark.skip("Closeness calculations not quite working yet")
```

Comment:
pytest.mark.xfail instead?

Reply:
It only fails some of the time, because sometimes the random values are bad.
It would be better to change them to fixed "errors" to get deterministic behaviour.
Do you still thing xpass is more appropriate in this case?

Reply:
Ah, I see. The pytest docs seem to suggest that marking as xfail with strict=False is an appropriate way of marking a flaky test. An xfail mark will still allow the test to run.